sources/main.cpp

```diff
@@ -54,97 +57,82 @@ elapsed(const std::chrono::steady_clock::time_point& timePoint) {
 }
 
 static dfl::inputs::Configuration::SimulationKind
-getSimulationKind(const dfl::common::Options::RuntimeConfiguration& runtimeConfig) {
-  if (runtimeConfig.contingenciesFilePath.empty()) {
+getSimulationKind(const dfl::common::Options::Request& req) {
+  switch (req) {
+  case dfl::common::Options::Request::RUN_SIMULATION_N:
     return dfl::inputs::Configuration::SimulationKind::STEADY_STATE_CALCULATION;
-  } else {
+  case dfl::common::Options::Request::RUN_SIMULATION_SA:
     return dfl::inputs::Configuration::SimulationKind::SECURITY_ANALYSIS;
+  case dfl::common::Options::Request::RUN_SIMULATION_NSA:
+    return dfl::inputs::Configuration::SimulationKind::STEADY_STATE_CALCULATION_AND_SECURITY_ANALYSIS;
```

Comment:
I think that we can remove dfl::inputs::Configuration::SimulationKind::STEADY_STATE_CALCULATION_AND_SECURITY_ANALYSIS and test directly the runtimeConfig in the main